WordPress/src/jetpack/assets/support_chat_widget.html

```diff
@@ -1,21 +1,9 @@
 <html>
-<head>
-    <script type="text/javascript" src="/assets/support_chat_widget.js"></script>
-</head>
-<body>
-<script type="text/javascript">
-    DocsBotAI.init({
-            id: 'TqTdebbGjJeUjrmBIFjh/YbAMwiheXLs2Ue5j7elH',
-            options: {
-                horizontalMargin: 40,
-                verticalMargin: 60,
-            },
-        }).then(() => {
-            // Safely do stuff here after the widget is loaded.
-        })
-        setTimeout(() => {
-           DocsBotAI.open()
-        }, 200);
-    </script>
-</body>
+    <head>
+        <script type="text/javascript" src="/assets/bot_widget.js"></script>
+    </head>
+    <body>
+        <script type="text/javascript" src="/assets/support_chat_widget.js"></script>
+    </body>
```

Comment:
Why are there two scripts for the bot here, `bot_widget.js` and `support_chat_widget.js`? I guess you were trying something, and we'll remove one of these, right?